src/config/edge/manifest.json

```diff
@@ -0,0 +1,41 @@
+{
+    "name": "Octotree",
+    "version": "$VERSION",
+    "manifest_version": 2,
+    "author": "Buu Nguyen",
+    "description": "Code tree for GitHub",
+    "icons": {
```

Comment:
Which manifest file Edge uses? Why icons are defined in both files?

Reply:
Actually, Edge extension is packaged as `appx` and published to Microsoft Store. Both manifests are used.

* `manifest.json` is used by Edge and the icon is displayed within Edge;
* `AppxManifest.xml` is used by Microsoft Store and the icon is displayed in the Marketplace.

They should be the same icon but size requirement of them are not the same.